google/cloud/bigtable/tests/data_integration_test.cc

```diff
@@ -598,41 +561,71 @@ TEST_F(DataIntegrationTest, TableReadMultipleCellsBigValue) {
     return;
   }
 
-  std::string const table_id = RandomTableId();
-  auto table = CreateTable(table_id, table_config);
+  // TODO(#2015) - Capture the elapsed time for each operation, so we can figure
+  //   out which one causes the test to timeout.
+  auto start = std::chrono::steady_clock::now();
+  auto elapsed_ms = [&start] {
+    using ms = std::chrono::milliseconds;
+    auto now = std::chrono::steady_clock::now();
+    auto elapsed = now - start;
+    start = now;
+    return std::chrono::duration_cast<ms>(elapsed).count();
+  };
+
+  auto table = GetTable();
+  // TODO(#2015) - Capture elapsed time to troubleshoot flakiness.
+  SCOPED_TRACE("CreateTable() " + std::to_string(elapsed_ms()) + "ms");
 
   std::string const row_key = "row-key-1";
-  // cell vector contains 10 cells of 25 MiB
-  auto const MiB = 1024L * 1024L;
-
-  std::string value((25 * MiB), 'a');
+  // cell vector contains 4 cells of 32 MiB each, or 128 MiB (without
+  // considering any overhead). That is much larger that the default gRPC
+  // message size (~4 MiB), and yet much smaller than the configured message
```

Comment:
I believe the max was changed to 256MiB. Is this only true for the emulator?

Reply:
The default from gRPC is 4MiB.  Our default is 256MiB. That comment refers to the original gRPC default.